pkg/util/validation/validate.go

```diff
@@ -14,14 +16,16 @@ import (
 const (
 	discardReasonLabel = "reason"
 
-	errMissingMetricName = "sample missing metric name"
-	errInvalidMetricName = "sample invalid metric name: %.200q"
-	errInvalidLabel      = "sample invalid label: %.200q metric %.200q"
-	errLabelNameTooLong  = "label name too long: %.200q metric %.200q"
-	errLabelValueTooLong = "label value too long: %.200q metric %.200q"
-	errTooManyLabels     = "sample for '%s' has %d label names; limit %d"
-	errTooOld            = "sample for '%s' has timestamp too old: %d"
-	errTooNew            = "sample for '%s' has timestamp too new: %d"
+	errMissingMetricName  = "sample missing metric name"
+	errInvalidMetricName  = "sample invalid metric name: %.200q"
+	errInvalidLabel       = "sample invalid label: %.200q metric %.200q"
+	errLabelNameTooLong   = "label name too long: %.200q metric %.200q"
+	errLabelValueTooLong  = "label value too long: %.200q metric %.200q"
+	errTooManyLabels      = "sample for '%s' has %d label names; limit %d"
+	errTooOld             = "sample for '%s' has timestamp too old: %d"
+	errTooNew             = "sample for '%s' has timestamp too new: %d"
+	errDuplicateLabelName = "duplicate label name: %.200q metric %.200q"
```

Comment:
`duplicate` > `duplicated`

_same comment applies below_

Reply:
I think both work fine here. Duplicate is an adjective, so duplicate label is valid expression. I believe :) ... but given that english is a second language for me, I can be convinced.

Reply:
Duplicate and Duplicated both work and I kinda prefer Duplicate :)